experimental/addons/istio-gateway/template.cue

```diff
@@ -0,0 +1,98 @@
+output: {
+	apiVersion: "core.oam.dev/v1beta1"
+	kind:       "Application"
+	metadata:
+		name: 	   "istio-gateway"
+		namespace: "vela-system"
+	spec: {
+		components: [{
+			type: "k8s-objects"
+			name: "ns-istio-system"
+			properties: objects: [{
+				apiVersion: "v1"
+				kind:       "Namespace"
+				metadata: name: "istio-system"
+			}]
+		}, {
+			type: "helm"
+			name: "base"
+			properties: {
+				repoType:        "helm"
+				url:             "https://istio-release.storage.googleapis.com/charts"
+				chart:           "base"
+				version:         "1.14.2"
+				targetNamespace: "istio-system"
+			}
+		}, {
+			type: "helm"
+			name: "istiod"
```

Comment:
Is this component required?

Reply:
Yes, istiod provides istio's control plane.